README.md

```diff
@@ -62,6 +62,10 @@ How to [secure your setup](/doc/security/ssl.md).
 - Electron and ChromeOS applications to bridge the gap between local<->remote.
 - Run VS Code unit tests against our builds to ensure features work as expected.
 
+### Notes
+
+- At the moment we can't use the official VSCode Marketplace. Coder has created a collection of extensions that are open-source. While you can't directly download extensions from the official VSCode Marketplace, you can upload them to be installed in code-server.
```

Comment:
Might be better to phrase this as:
`We've created a custom extension marketplace focused around open-sourced extensions.`
I'd like to convey that we don't just package a few additional exts, and it is a full marketplace that will be improved.